src/index.tsx

```diff
@@ -16,6 +16,8 @@ type Props = {
   containerClassName?: string;
   inputClassName?: string;
   isPassword?: boolean;
+  placeholderCharacter?: string & { length: 1 };
+  showPlaceholder: false;
```

Comment:
I think we can handle the display of the placeholder using the `placeholderCharacter` prop, or do you see any case where we pass it and don't want to display it? Also, I would rename the prop to only `placeholder`

Reply:
I think I understand the motivation for this feature and I can see how it is valuable, but in terms of implementation, we can just have a `placeholder` prop and nothing else, if it is set, we show the placeholder otherwise we don't. Basically this: 


```jsx
type Props = {
  allowedCharacters?: typeof allowedCharactersValues[number];
  ...
  placeholder?: string & { length: 1 };
}

...

<input
  ...
  placeholder={placeholder}
/>
```

Reply:
It's perfect!!